libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -666,3 +666,12 @@ ebpf_query_time_since_boot(bool include_suspended_time)
         return KeQueryInterruptTimePrecise(&qpc_time);
     }
 }
+
+ebpf_result_t
+ebpf_guid_create(_Out_ GUID* new_guid)
+{
+    if (NT_SUCCESS(ExUuidCreate(new_guid)))
```

Comment:
See earlier comments.  https://datatracker.ietf.org/doc/html/rfc4122 section 4 explains that there are many ways to create a UUID. It would be more practical/secure to use a name-based UUID (section 4.3) rather than asking ExUuidCreate to create a random UUID (section 4.4).

Reply:
Agreed. The eventual goal is to eliminate the use of random module IDs in favor of using either something concrete (attach type, program ID etc), but that is outside of the scope of this PR.